spec/models/scheduled_message_spec.rb

```diff
@@ -12,4 +12,13 @@
      it { should validate_presence_of(:time_of_day) }
      it { should validate_presence_of(:title) }
   end
+
+  describe '.date_time_ordering' do
+    it 'should display the messages in chronological order' do
+      m1 = create(:scheduled_message, time_of_day: '2000-01-01 16:00:00 UTC')
+      m2 = create(:scheduled_message, time_of_day: '2000-01-01 14:00:00 UTC')
+      m3 = create(:scheduled_message, time_of_day: '2000-01-01 11:00:00 UTC')
+      expect(ScheduledMessage.date_time_ordering).to match_array([m1, m2, m3])
```

Comment:
perhaps we should add another scheduled message for a different day here to make sure date AND time ordering are working?